### ISSUE TITLE
question about simclr1 results

### ISSUE THREAD

**Issue by homelifes** [google-research/simclr#124]
Hi. May I know the cifar results on page 18 of SimCLR1 paper, the batch sizes and epochs in the graph refer to pre-training or linear evaluation? For example, 100 epochs with a batch size of 1024 achieves a linear evaluation accuracy of 85%. That means you pre-train with a batch size of 1024 for 100 epochs, or you linearly evaluate (after pre-training) with a batch size of 1024 for 100 epochs?

**Comment by chentingpc** [comment]
These are the pretraining hparams. The linear eval can be obtained by training linear classifier on-the-fly during the pretraining with a stop-gradient operator to prevent leaking of supervised labels. See [here](https://github.com/google-research/simclr/blob/3ad6700c1b139ee18e43f73546b7263a710de699/tf2/model.py#L272) for the code.

**Comment by homelifes** [comment]
Thanks!
